packages/react/src/UnderlineNav/UnderlineNav.tsx

```diff
@@ -386,9 +386,10 @@ export const UnderlineNav = forwardRef(
                     }
 
                     return (
-                      <ActionList.LinkItem
+                      <ActionList.Item
                         key={menuItemChildren}
                         style={menuItemStyles}
+                        /* eslint-disable-next-line primer-react/prefer-action-list-item-onselect */
```

Comment:
(non blocker) worth asking @pksjce or @broccolinisoup if there is a reason to prefer onClick over onSelect or if we can use onSelect here